runtime/browser/xwalk_browser_context.cc

```diff
@@ -349,6 +315,17 @@ net::URLRequestContextGetter*
 #endif
 }
 
+net::URLRequestContextGetter* XWalkBrowserContext::CreateMediaRequestContext() {
```

Comment:
No of course he did not meant to remove the override. He meant to keep the the code that was in GetMediaRequestContextForStoragePartition and put it in CreateMediaRequestContextForStoragePartition just like upstream did with ProfileImpl.